ansible/clone_fms.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Clone the FMS GitHub repo
+  hosts: remote_host
+  gather_facts: true
+  become: yes
```

Comment:
for my own understanding, what does `become: yes` do here? does it work without become yes, since nothing in the current task list needs sudo?